yarn.lock

```diff
@@ -8411,7 +8411,7 @@ react-dom@16.8.6:
     loose-envify "^1.1.0"
     object-assign "^4.1.1"
     prop-types "^15.6.2"
-    scheduler "^0.19.1"
+    scheduler "^0.13.6"
```

Comment:
This lib keeps getting changed, but I'm not quite sure what is triggering it.

Reply:
Weird. I'm not seeing that on my local yarn.lock. I'm seeing 0.13.6

Reply:
What is the correct version @thestephenmarshall